.circleci/config.yml

```
@@ -640,6 +640,9 @@ jobs:
       - run-tests-linux:
           test_targets: "wasm2js1"
   test-wasm64:
+    environment:
+      # Only run 2 tests at a time to avoid OOM (since each tests used >4gb)
```

```suggestion
      # Only run 2 tests at a time to avoid OOM (since each test uses >4gb)
```
Can this be in a separate PR? This OOM fix makes sense though.

---

Do we have data that shows we OOM btw?